nuxeo-platform-video-convert/src/main/java/org/nuxeo/ecm/platform/video/convert/BaseVideoConversionConverter.java

```diff
@@ -82,6 +86,7 @@
         }
         // delete the file as we need only the path for ffmpeg
         outFile.delete();
+        Framework.trackFile(outFile, this);
```

Comment:
`this` is not the proper marker to use to track the file, its lifespan is too long (it lives in the conversion service as long as Nuxeo is running). You want as a marker something that's freed when the file is not needed anymore.